### ISSUE TITLE
Adjust fake capacity to have nvidia.com~1accelerator-memory

### ISSUE THREAD

**Issue by harche** [openshift/instaslice-operator#46]
The current fake capacity needs to be patched to make sure emulated demo works fine. 

```bash
kubectl patch node <your-node-name> --subresource=status --type=json -p='[{"op":"add","path":"/status/capacity/nvidia.com~1accelerator-memory","value":"80Gi"}]'
```

**Comment by asm582** [comment]
Solved by #47

**Comment by harche** [comment]
fixed by https://github.com/openshift/instaslice-operator/pull/47
